docker-compose.yml

```diff
@@ -11,4 +11,19 @@ services:
     ports:
       - ${DB_PORT}:5432
     volumes:
-      - ./docker-data/db:/var/lib/postgresql/data
+      - pgdata:/var/lib/postgresql/data
```

Comment:
Why did you change this?

Reply:
> Why did you change this?


Generally speaking there is not much difference but I decided to use a named volume to allow it to be shared with other, future, containers.
Relative volume paths are generally used when you want to sync a folder in your directory with the docker container and may cause some issues on deploying to a server, a bitter description is in this [stackoverflow question](https://stackoverflow.com/questions/46907558/docker-compose-relative-paths-vs-docker-volume#:~:text=A%20reason%20you,with%20relative%20paths.)

Reply:
You should have added this to the "What is the new behaviour?" section and your reasoning.

@w3cj will have to decide if he wants a named volume.

Reply:
My bad!